android/src/main/java/com/applovin/reactnative/AppLovinMAXAdView.java

```diff
@@ -17,6 +22,8 @@
 class AppLovinMAXAdView
         extends ReactViewGroup
 {
+    private static final Map<String, Map<MaxAdFormat, Set<MaxAdView>>> mAdViews = new HashMap<>( 2 );
```

Comment:
?? a data structure of a data structure of another data structure - this is not readable or good convention - can you explain what this is then I can offer suggestions.

Reply:
This is trying to save a list of MaxAdView with both adUnitId and adFormat as a key, like this in json.

```
{
   "adUnitID1":{
      "adFormat1":[
         MaxAdView111,
         MaxAdView112,
         MaxAdView113
      ],
      "adFormat2":[
         MaxAdView121,
         MaxAdView122,
         MaxAdView123
      ]
   },
   "adUnitID2":{
      "adFormat1":[
         MaxAdView211,
         MaxAdView212,
         MaxAdView213
      ],
      "adFormat2":[
         MaxAdView221,
         MaxAdView222,
         MaxAdView223
      ]
   }
}
```

The list is preferred to be an ordered set, so that we can reuse it from the oldest.  How can we do this?